### ISSUE TITLE
Send http query to search results

### ISSUE THREAD

**Issue by PeterVos111** [SharePoint/sp-dev-solutions#317]
#146 ## Category
- [x] Question
- [ ] Bug
- [ ] Enhancement

If it possible to send a query directly to the searchresult webpart? I can send a 
                'q=<query>'  
to the default SP Search, but i want the query to be send to the searchresult webpart, with the ability to use firther refiners.

Thanks in advance

**Comment by wobba** [comment]
Yes it's possible and you configure the search result webpart to consume the query from that param. Add q=lala in the param before editing the page, and you should be able to set it up.

![image](https://user-images.githubusercontent.com/51104/66994411-7f869880-f0cd-11e9-91a9-4a9d9b926d25.png)

**Comment by PeterVos111** [comment]
That works great, Except i would like my Searchresults to use both the query param, and the searchbox query.
OOTB i can only connect to one source

**Comment by wobba** [comment]
@PeterVos111 so you need a query variable to pick up params like classic supports. Feel free to add that as a feature request :)

**Comment by PeterVos111** [comment]
Thanks for the tip. I did just that :)
